tests/test_resources/test_modules/test_folder.py

```diff
@@ -17,6 +17,19 @@ def fs_str_rh_fn(folder):
 
 
 class TestFolder(tests.test_resources.test_resource.TestResource):
+
+    MAP_FIXTURES = {"resource": "folder"}
+
+    UNIT = {"folder": ["s3_folder", "gcs_folder"]}
+    LOCAL = {"folder": ["s3_folder", "gcs_folder"]}
```

Comment:
local should include testing folder interaction with a system of our local test docker container. either creating a fixture for some `local_cluster_folder`, or further parameterizing this with a cluster argument as well. And local is run on CI so until we get support for s3 and gcs there, those should remain in minimal up